AirCasting/Settings/SettingsView.swift

```diff
@@ -9,92 +9,136 @@ import SwiftUI
 import Foundation
 
 struct SettingsView: View {
+    
     let logoutController: LogoutController
-    @State private var isToggle : Bool = false
+    @State private var isToggle: Bool = false
     @State private var showModal = false
+    
     var body: some View {
         NavigationView {
             List {
                 Section() {
-                    NavigationLink(destination: MyAccountViewSingOut()) {
-                        Text("My Account")
-                    }
+                    signOutLink
                 }
                 
                 Section() {
                     VStack(alignment: .leading) {
                         HStack(spacing: 5) {
-                            Text("Contribute to CrowdMap")
-                            Toggle(isOn: $isToggle){
-                                Text("Switch")
-                                    .font(.title)
-                                    .foregroundColor(Color.white)
-                            }
-                            .toggleStyle(SwitchToggleStyle(tint: .accentColor))
+                            crowdMapTitle
+                            crowdMapSwitch
                         }
                         Spacer()
-                        Text("Data contributed to the CrowdMap is publicly available at aircasting.org")
-                            .fontWeight(.light)
+                        crowdMapDescription
                     }
-                    
-                    Button(action: {
-                        showModal.toggle()
-                    }) {
-                        Group {
-                            HStack {
-                                Text("Backend settings")
-                                    .accentColor(.black)
-                                Spacer()
-                                Image(systemName: "control")
-                                    .accentColor(.gray).opacity(0.6)
-                            }
-                        }
-                    }.sheet(isPresented: $showModal, content: {
-                        BackendSettingsModalView()
-                    })
+                    showModalButton
```

Comment:
good job on dividing stuff over here. One thing that can be better is this name. "showModalButton" is not really carrying any intent description. 'Modal' is just a presentation style, so showModalButton is saying nothing. Maybe lets think of a better name (from what I see here this button navigates the user to some backend settings, so maybe something like `navigateToBackendSettingsButton` ? )